test/integration/progressesTasks.test.js

```diff
@@ -516,4 +516,104 @@ describe("Test Progress Updates API for Tasks", function () {
         });
     });
   });
+
+  describe("GET /progresses (getPaginatedProgressDocument)", function () {
+    let userId;
+    let taskObject1;
+    let taskId1;
+
+    beforeEach(async function () {
+      userId = await addUser(userData[1]);
+      taskObject1 = await tasks.updateTask(taskData[0]);
+      taskId1 = taskObject1.taskId;
+      const progressData1 = stubbedModelTaskProgressData(userId, taskId1, 1683626400000, 1683590400000); // 2023-05-09
+      const progressData2 = stubbedModelTaskProgressData(userId, taskId1, 1683885600000, 1683849600000); // 2023-05-12
+      await firestore.collection("progresses").doc("taskProgressDocument1").set(progressData1);
+      await firestore.collection("progresses").doc("taskProgressDocument2").set(progressData2);
+    });
```

Comment:
We are not clearing the data in afterEach hook?

Reply:
Tthe cleanDb() function is already being called in the afterEach hook of the parent describe block, which ensures the database is cleaned after every test